README.md

```diff
@@ -11,3 +11,79 @@ Replicated.
 
 Contributions are greatly appreciated. We are currently evolving our
 [contributors guide](Contributing.md) to better support your contributions.
+
+
+## How to setup Compatibility Testing in GitHub Actions
```

Comment:
We should document this in the patterns. See - https://github.com/replicatedhq/platform-examples/tree/main/patterns

Reply:
I have moved it to the patterns folder